dbt_project.yml

```
@@ -1,24 +1,34 @@
 config-version: 2
 
 name: 'zendesk_source'
-version: '0.0.1'
+version: '0.1.2'
```

```suggestion
version: '0.2.0'
```

let's upgrade to 0.2.0 as there are significant changes in this

---

Updated